pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "co2calculator"
+version = "0.0.1"
+license = "GPL-3.0-only"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
Suggestion: We could add `authors = ["pledge4future Team <info@pledge4future.org>"]` and individual names who ever wishes to?